ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -102,6 +102,49 @@ internal sealed class ArchiWebHandler : IDisposable {
 			return await UrlPostWithSession(SteamCommunityURL, request, data, referer).ConfigureAwait(false);
 		}
 
+		internal async Task AcceptDigitalGiftCards() {
+			const string requestGifts = SteamStoreURL + "/gifts";
+			HtmlDocument response = (await WebLimitRequest(SteamStoreURL, async () => await WebBrowser.UrlGetToHtmlDocument(requestGifts).ConfigureAwait(false)).ConfigureAwait(false))?.Content;
+			if (response == null) {
+				Bot.ArchiLogger.LogNullError(nameof(response));
+				return;
+			}
+
+			const string requestRedeemGift = "/gifts/0/resolvegiftcard";
+			HtmlNodeCollection nodes = response.DocumentNode.SelectNodes("//div[starts-with(@id,'pending_gift_')]");
+			foreach (string strGID in nodes.Where(node => node?.ChildNodes?.Any(giftNode => giftNode?.Attributes["class"]?.Value?.StartsWith("pending_giftcard") == true) == true).Select(node => node.Attributes["id"]?.Value?.Substring(13))) {
+				if (string.IsNullOrEmpty(strGID)) {
+					Bot.ArchiLogger.LogNullError(nameof(strGID));
+					continue;
```

Comment:
Every fatal error that is supposed to be corrected by ASF devs should end with `return`, not a `continue`.